iceoryx_posh/include/iceoryx_posh/iceoryx_posh_types.hpp

```diff
@@ -100,14 +100,12 @@ namespace popo
 using WaitSetIsConditionSatisfiedCallback = cxx::ConstMethodCallback<bool>;
 }
 constexpr uint32_t MAX_NUMBER_OF_CONDITION_VARIABLES = 1024U;
```

Comment:
I'm wondering if this should also be configurable?

Reply:
Maybe not a bad idea. This restricts the user on how many waitset/listeners can be created in an iceoryx environment

Reply:
But I would keep this for iceoryx 3.0 and for the iceoryx posh types refactoring.